jest.config.js

```diff
@@ -1,8 +1,18 @@
-const supportsESM = parseInt(process.versions.node) >= 12;
+const semver = require("semver");
+const nodeVersion = process.versions.node;
+const supportsESMAndJestLightRunner = semver.satisfies(
+  nodeVersion,
+  // ^12.22 || >=14.17 : Node will throw "t.isIdentifier is not a function" when test is running in worker threads.
+  // ^13.7: `resolve.exports` specifies conditional exports in package.json
+  "^12.22 || ^13.7 || >=14.17"
```

Comment:
In Babel 8 I'd like to unify our test runner to only use the light runner (so that we don't have to maintain compatibility with two parallel runners). We should probably just drop 14.15 and 14.16: they are quite old, and there are different security issues with them anyway (LTS versions are so that users can easily upgrade within the same major, not so that they can avoid upgrading).

Reply:
I agree. Plus node 14.17 comes with top level await. I would rather bump Babel 8 versions to "every node versions released after 2022-05-01"

Reply:
That might be too much (people often don't update immediately), but we might do something "released in 2022", or "released in the last 6 months before Babel 8.0.0" for Node.js 14 and 16.